be/tsconfig.json

```diff
@@ -6,16 +6,20 @@
     "moduleResolution": "node",
     "esModuleInterop": true,
     "strict": true,
+    "strictPropertyInitialization": false,
+    "strictNullChecks": false,
     "sourceMap": false,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "rootDir": "./",
+    "rootDir": "./src",
     "baseUrl": "./",
     "types": ["jest"],
     "outDir": "dist/"
   },
+  "include": ["src/**/*.ts"],
```

Comment:
Is this still required? The `rootDir` has been updated to `./src`...

Reply:
Has been removed